### ISSUE TITLE
Revise and reorganize Federalist support site (federalist-docs.18f.gov)

### ISSUE THREAD

**Issue by thebestsophist** [18F/federalist.18f.gov#212]
#### User Story

As a user or possible user of Federalist, I need to easily find information that supports me as I learn about how Federalist works and how I should use it.

#### Background (Optional)

Federalist-docs.18f.gov serves as the support guide for Federalist. We need to streamline content, reorganize the content structure so that it aligns with [our new IA](https://app.mural.ly/t/gsa6/m/gsa6/1519835839589/1be43108db39f31a70854a9cb515091aed990312) for the site, refine navigation, and add some new content. New sections or high-level pages should include (with details for each section outlined in the IA):

- Support Home
- Get started
- Your site
- Templates
- Integrations
- Security and compliance
- For content creators 
- For designers and developers
- Contribute

#### Acceptance Criteria

- [ ] The Federalist support site reflects the structure of [our IA](https://app.mural.co/t/gsa6/m/gsa6/1522093516875/88001f9a465e1c1a57860657f3cbf6cb4708a7a4)
- [ ] Support content lives on the support site, while marketing content lives at federalist.18f.gov

_After evaluating, edit this part:_

#### Implementation outline (if higher than "low" effort):

- [ ] <TODO>

**Comment by ShannonBad** [comment]
Related to #165

**Comment by wslack** [comment]
I closed the other issue but one of the earlier suggestions was splitting the docs site into a "user focused" section and a separate" dev focused" section.

**Comment by thebestsophist** [comment]
Possible reorganization in the bottom right of here: https://app.mural.ly/t/gsa6/m/gsa6/1522093516875/88001f9a465e1c1a57860657f3cbf6cb4708a7a4

**Comment by annahsebok** [comment]
Updated issue to reflect our new [content approach](https://docs.google.com/document/d/1V1RPQ3fFdclCDE2ZCm6Htt25DLcPLJ5mbE01UqnnvnI/edit#).

**Comment by wslack** [comment]
Deconverted form Epic

**Comment by wslack** [comment]
Dropping in priority, putting in the new template as well.

**Comment by annahsebok** [comment]
@wslack I've updated this issue to reflect the need for implementation of the new IA. I'm leaving the outline of the implementation steps for @coreycaitlin since I don't want to dictate how she decides to implement. The core purpose of this work is to reorganize the existing federalist-docs site into more of a support site, organized by the approach we identified in [the IA](https://app.mural.co/t/gsa6/m/gsa6/1522093516875/88001f9a465e1c1a57860657f3cbf6cb4708a7a4).

I'd also strongly recommend that this get moved up in the backlog between #1854 and #1856. This should definitely be done before #1856 (validation testing of the new IA once it's been implemented).
